react/features/base/connection/actions.any.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'lodash';
+
+import {
+    appendURLParam,
+    getBackendSafeRoomName,
+    parseURIString
+} from '../util';
+
+import logger from './logger';
+
+/**
+ * Constructs options to be passed to the constructor of {@code JitsiConnection}
+ * based on the redux state.
+ *
+ * @param {Object} state - The redux state.
+ * @param {boolean} mobile - Whether we use it for mobile, whether to try enabling ab testing of websockets.
```

Comment:
You don't need the parameter, you can check if `navigator.product === 'ReactNative'` to detect mobile without extra dependencies.

Reply:
Dropped the param.